backend/database/versions/9023d0bf61ab_harmonize_schema_state_prod.py

```diff
@@ -0,0 +1,86 @@
+"""harmonize_schema_state_prod
+
+Revision ID: 9023d0bf61ab
+Revises: d9fb29897244
+Create Date: 2020-08-28 15:00:56.262160
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "9023d0bf61ab"
+down_revision = "d9fb29897244"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table("dataset_deployment")
+    op.alter_column("dataset", "project_id", existing_type=sa.VARCHAR(), nullable=False)
+    op.alter_column("dataset", "project_status", existing_type=sa.VARCHAR(), nullable=False)
+    op.alter_column("dataset", "revision", existing_type=sa.INTEGER(), nullable=True)
```

Comment:
Why is revision nullable?

Reply:
Because [this](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/corpora/common/corpora_orm.py#L241) doesn't have a `nullable = False` kwarg. I agree that it should be non null though. Maybe that's a good candidate for a first post-harmonization migration?